packages/rest-generator/package.json

```diff
@@ -33,7 +33,7 @@
     "check:dependencies": "depcheck . --ignore-bin-package=true"
   },
   "dependencies": {
-    "@openapitools/openapi-generator-cli": "cli-4.2.1",
+    "@openapitools/openapi-generator-cli": "^1.0.18-4.3.1",
```

Comment:
I'm afraid of making dynamic version for this. But we should have tests for protection.

Reply:
Why are you afraid? Did we have some issues?